### DISCUSSION TITLE
Is it possible to let 2 apps use the same asynstorage?

### DISCUSSION THREAD

**Discussion by VerjansRobbe** [react-native-async-storage/async-storage#758] [Q&A]
Is it possible to let 2 apps use the same asynstorage? I have a strange situation, I need 2 apps  that i both have made to access the same asyncstorage. Or some other type of shared storage. Just passing data or sending it to a remote database will not be possible because it will be quite large.

Thanks in advance.

**Comment by krizzu** [comment]
Each app is sandboxed, so they have no access to other app data. 
One way to share data is to create a file, save data there and then pass the file location to other app, so it can read it. But that's outside of AsyncStorage scope
